### ISSUE TITLE
Restore `rb` and `rtc`

### ISSUE THREAD

**Issue by frivoal** [w3c/html-aam#602]
https://github.com/w3c/html-aam/pull/253 removed the `rb` and `rtc` sub elements of ruby, because they are been marked obsolete in the HTML specification. https://www.w3.org/TR/html-ruby-extensions/ is restoring them, and so they should be restored in accessibility mappings as well.

(See the [introduction](https://www.w3.org/TR/html-ruby-extensions/#intro) and [appendix B](https://www.w3.org/TR/html-ruby-extensions/#diff-html) for rationale.)

In terms of roles, I believe the following ones would be correct:
* for `rb`
  * MSAA+ IAccessible2: Role: ROLE_SYSTEM_TEXT; IA2_ROLE_TEXT_FRAME
  * AX: AXRubyBase

* for `rtc`: it's container for `rt` elements, and occasionally anonymous text nodes with the same semantics as rt, so probably the same roles as `rt` would seem to make sense.

This blocks https://github.com/w3c/html-ruby/pull/45

**Comment by spectranaut** [comment]
Just for reference: https://github.com/w3c/html-aria/pull/572
